filebeat/filebeat.full.yml

```diff
@@ -490,6 +490,19 @@ filebeat.prospectors:
 #- add_locale:
 #    format: offset
 #
+# The following example enriches each event with docker metadata, it matches
+# given fields to an existing container id and adds info from that container:
+#
+#processors:
+#- add_docker_metadata:
+#    match_fields: ["system.process.cgroup.id"]
```

Comment:
So the config option is kind of `field_that_matches_container_id: ...`. What is the use case of having multiple fields defined? 

For the cgroup docker part: I see that this is what it's doing, but why would someone then not just use the docker module instead as he will rely on the docker api.